README.md

```diff
@@ -386,10 +390,16 @@ npx code-push update-history --binary-version 1.0.0 --app-version 1.0.1 \
 Create a CodePush bundle file.
 
 **Example:**
-```
+```bash
 npx code-push bundle --platform android --entry-file index.js
+
+# Expo project
+npx code-push bundle --type expo --platform android --entry-file index.js
 ```
+- `--type`: Project type (react-native (default) | expo)
 
 By default, the bundle file is created in the `/build/bundleOutput` directory.
 
+>[!NOTE] For Expo projects, the CLI uses `expo export:embed` command for bundling instead of React Native's bundle command.
```

Comment:
```suggestion
> [!NOTE]
> For Expo projects, the CLI uses `expo export:embed` command for bundling instead of React Native's bundle command.
```

It'll be rendered like this

> [!NOTE]
> For Expo projects, the CLI uses `expo export:embed` command for bundling instead of React Native's bundle command.